learning-ojs/en/editorial-workflow.md

```diff
@@ -561,6 +561,10 @@ This will open a new window.
 
 ![](./assets/learning-ojs-3-ed-accept.png)
 
+Note that if the journal has enabled an article processing charge (APC) to be charged to authors, the option will appear at this stage to notify the author that the payment is due. Selecting "Request publication fee" will prompt a payment notification email to be sent to the author with payment instructions. For information about enabling author fees, please see the [Subscriptions chapter > Payment Types](./subscriptions) and [Distribution settings > Enable Payment](./settings-distribution).
+
+![](./assets/learning-ojs-3-request-fee.png)
```

Comment:
@kshuttle I don't think the file has been uploaded yet, could you add it? You should be able to upload in the branch for this pull request here: https://github.com/kshuttle/pkp-docs/tree/patch-8/learning-ojs/en/assets